k8s/mariadb-galera/README.md

```diff
@@ -104,7 +102,7 @@ export APP_INSTANCE_NAME=mariadb-galera-1
 export NAMESPACE=default
 ```
 
-(Optional) Enable Stackdriver Metrics Exporter:
+#### (Optional) Enable Stackdriver Metrics Exporter:
```

Comment:
I think this should be just a plain text instead of a header. The following sections are the same important as this on, however the are plain text.

![image](https://user-images.githubusercontent.com/13849267/68380920-0836a880-0151-11ea-8b8e-0637b3ddb868.png)

Reply:
I rearranced to address this - let me know what you think!